usaspending_api/etl/es_transaction_template.json

```diff
@@ -286,9 +371,25 @@
             }
           }
         },
-        "type": {
-          "type": "keyword",
-          "null_value": "NULL"
+        "treasury_account_identifiers": {
```

Comment:
we talked about changing the TAS search so we didn't have to hit the database when doing that search, I don't know if you want to to include that here, but it would be like
`"treasury_accounts": {
          "type": "nested",
          "properties": {
            "aid": {
              "type": "integer"
            },
            "ata": {
              "type": "integer"
            },
            "main": {
              "type": "integer"
            },
            "sub": {
              "type": "integer"
            },
            "bpoa": {
              "type": "integer"
            },
            "epoa": {
              "type": "integer"
            },
            "a": {
              "type": "text"
            }
          }
        }`
you would also have to change the delta view to pull the data, like 
`LEFT OUTER JOIN (
  SELECT
    faba.award_id,
    JSONB_AGG(
      JSONB_BUILD_OBJECT(
        'aid', taa.agency_id,
        'ata', taa.allocation_transfer_agency_id,
        'main', taa.main_account_code,
        'sub', taa.sub_account_code,
        'bpoa', taa.beginning_period_of_availability,
        'epoa', taa.beginning_period_of_availability,
        'a', taa.availability_type_code
      )
    ) treasury_account_identifiers
  FROM
    federal_account fa
    INNER JOIN treasury_appropriation_account taa ON fa.id = taa.federal_account_id
    INNER JOIN financial_accounts_by_awards faba ON taa.treasury_account_identifier = faba.treasury_account_id
  WHERE
    faba.award_id IS NOT NULL
  GROUP BY
    faba.award_id)
 TREASURY_ACCT ON (TREASURY_ACCT.award_id = UTM.award_id);`

Reply:
Will make the change. That escaped me when working on this, but it definitely should go with this PR.